storage/transfermanager/downloader.go

```diff
@@ -31,6 +31,10 @@ import (
 	"google.golang.org/api/iterator"
 )
 
+// ErrDownloaderClosed indicates that the Downloader has already been closed and
+// must not be used for new operations.
+var ErrDownloaderClosed = errors.New("transfermanager: Downloader used after WaitAndClose was called")
```

Comment:
I don't think this should be an exported sentinel error, it's not really something that people should need to check for programmatically.